lxd/identities.go

```diff
@@ -1126,6 +1131,7 @@ func getCurrentIdentityInfo(d *Daemon, r *http.Request) response.Response {
 		Identity:             *apiIdentity,
 		EffectiveGroups:      effectiveGroups,
 		EffectivePermissions: effectivePermissions,
+		IsFineGrained:        isFineGrained,
```

Comment:
Can we just set `identity.IsFineGrainedIdentityType(string(id.Type))` as the field value? Why does it need to be evaluated in the transaction?

Reply:
no reason :) fixed